armi/physics/neutronics/fissionProductModel/tests/test_fissionProductModel.py

```diff
@@ -75,32 +66,6 @@ def test_getAllFissionProductNames(self):
         self.assertIn("XE135", fissionProductNames)
 
 
-class TestFissionProductModelIndependentLumpedFissionProducts(unittest.TestCase):
```

Comment:
Wait, why is this removed now?

Reply:
It was unused code.

It had a `setUp()` method, but no tests. So no tests were ever run. Even the `setUp()` method was never run, which I proved to myself by putting an `assert False` in it.

Reply:
John ran code coverage on the unit tests to see which ones weren't running, and did some cleanup.

Reply:
(jinx)